powershell-gallery/docs-conceptual/powershellget/supported-repositories.md

```diff
@@ -142,9 +178,47 @@ GitHub Packages repositories:
   - The package is published as a private package. After publishing, you can use the GitHub
     interface to link it to the desired repository and change the visibility.
 
+### Publishing to GitHub Packages
+
+You can use the **Credential** or **ApiKey** parameter of the `Publish-PSResource` cmdlet to publish
+packages to a GitHub Package feed. You must create a personal access token (PAT) with the necessary
+scopes enabled. For more information on scopes and permissions, see
```

Comment:
We call out the specific scopes in the Azure Artifacts section, should we do the same here?

Reply:
The scopes are in the linked document. The scopes are more granular (independent). If you only want to write, you don't need delete. So I left it to the reader to decide.